README.md

```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
 
 **Note**: The recommended restart policy for the container is `on-failed` or `no`
 
+#### Docker Compose Example
+```yaml
+...
```

Comment:
I would just put `services:` here instead of the `...` (and at the end). It makes it easier to copy

Reply:
Addressed in: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/146194802d654287272dbfdf9e2a9187d74ee523